src/app/components/media/AudioMediaCard.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Player } from '@meedan/check-ui';
+import FormControl from '@material-ui/core/FormControl';
+import MenuItem from '@material-ui/core/MenuItem';
+import Select from '@material-ui/core/Select';
+import styled from 'styled-components';
+import { FormattedMessage } from 'react-intl';
+import AspectRatio from '../layout/AspectRatio';
+import { units } from '../../styles/js/shared';
+
+const StyledPlaybackRate = styled.div`
+  margin-top: ${units(2)};
+  position: absolute;
+  right: 0;
+`;
+
+const AudioMediaCard = (props) => {
```

Comment:
This is essentially identical to `VideoMediaCard`. Can we just rename it to `MediaPlayerCard` and use it for both video and audio?

Reply:
Done